src/views/apply/dashboard.html

```diff
@@ -1,257 +1,222 @@
 {% extends 'default.html' %}
```

Comment:
Please can you summarise what changes you've made to this file, as it's unclear from the single commit and the diff.

Reply:
- Consistent tabs
- Removed `<section>` tags
- Moving elements around and adding `<div>`s to work with the styling